server/routes/swipeRoutes.js

```diff
@@ -92,68 +92,73 @@ router.post("/nextuser/:userid", async (req, res) => {
     console.log(err);
   }
 });
-router.post("/recommendation/:userid/:count?", async (req, res) => {
+router.post("/recommendation", async (req, res) => {
   try {
-    const userid = req.params.userid;
-    const count = Number(req.params.count || 20);
+    const userid = req.body.userid;
+    const count = Number(req.body.count || 20);
 
     if (typeof count === "number") {
       const { listOfUsers, write, data } = await getRecommendations({
         userid,
         count,
       });
 
-      res.status(200).send({
+      res.status(200).json({
         message: `User ${userid} queue addition successful: added ${data.recommendqueue.length} recommendations`,
         data: write,
         added: listOfUsers,
       });
     } else {
-      res.status(400).send({ message: "Bad request" });
+      res.status(400).json({ message: "Bad request" });
     }
   } catch (err) {
     console.log(err);
   }
 });
 
-router.post("/:userid/:otherid/:bool", async (req, res) => {
+router.post("/swipe", async (req, res) => {
   try {
-    const bool = req.params.bool;
-    const userid = req.params.userid;
-    const otherid = req.params.otherid;
+    const userid = req.body.userid;
+    const otherid = req.body.otherid;
+    let bool = req.body.bool;
```

Comment:
I'm not entirely sure what `bool` is meant to represent. Perhaps you can give this a more descriptive name?